lib/index.js

```diff
@@ -31,7 +31,7 @@ var settings = [
   'partials',
   'pattern',
   'rename',
-  'exposeConsolidate'
+  'renderer',
```

Comment:
Well we have some differences in code style, and most of it is just personal preference I think. I personally don't like the aligned equals signs because it is an extra action that I otherwise don't have to do (and I use syntax highlighting so for me it's fine).

To consolidate these differences I think we should create an eslintrc file that we both agree on, and put it in it's own repo. Then we can use those rules for all our projects. Until then I would prefer sticking to the current code style, to prevent the code styles from drifting apart and becoming messy.

I'd suggest starting with the airbnb presets, and making changes from there where needed.